lib/app/client.js

```diff
@@ -140,6 +140,9 @@ async function loadAsyncComponents (to, from, next) {
     next()
   } catch (err) {
     err = err || {}
+    if (/^Loading chunk (\d)+ failed\./.test(err.message) && process.client) {
+      window.location.replace(location.href)
```

Comment:
With this improvement, we may make behavior more configurable. Sometimes persisting store data is needed for example before redirect or something like that.

My samplet suggestion is something like scroll behavior. But serialization is not actually a flexible idea. We can think about possible ways.

Reply:
Ye, making it configurable sounds fine. I could make it same way as scrollBehaviour if there no better idea.